imgui.ini

```diff
@@ -21,14 +21,14 @@ Collapsed=0
 DockId=0x00000008,0

```

Comment:
Are you sure you want to submit this file?

Reply:
Yep! I removed some windows and added a new one. So I need to update the layout.